crypto/src/openssl.rs

```diff
@@ -41,23 +44,37 @@ impl Hasher for OpensslHasher {
     }
 }
 
+// Currently only supports one CDI handle but in the future we may want to support multiple.
+const MAX_CDI_HANDLES: usize = 1;
+type ExportedCdiHandle = [u8; MAX_EXPORTED_CDI_SIZE];
```

Comment:
Should this be defined in lib.rs?

Reply:
Updated. I kept `MAX_CDI_HANDLES` private since I am not sure it belongs in the public API but I am happy to make it public (Not sure how much should be shared between this and the caliptra-sw implementation).